diff --git a/odlvpn2bgpd.py b/odlvpn2bgpd.py
index 338e81f..d1dea2f 100644
--- a/odlvpn2bgpd.py
+++ b/odlvpn2bgpd.py
@@ -17,7 +17,7 @@ import bgp_capnp
 # dirn = full path to this script
 #
 default_bgp_cfg = os.path.join(dirn, 'bgpd.conf')
-default_bgpd = '../quagga/bgpd/bgpd'
+default_bgpd = '/usr/lib/quagga/bgpd'
 default_thrift = os.path.join(dirn, 'vpnservice.thrift')
 #
 # end default settings
