diff --git a/setup.py b/setup.py
index bec846d..f883c13 100644
--- a/setup.py
+++ b/setup.py
@@ -32,11 +32,12 @@ dev_requires = [
 
 
 # cython detection
-try:
-    from Cython.Build import cythonize
-    CYTHON = True
-except ImportError:
-    CYTHON = False
+#try:
+#    from Cython.Build import cythonize
+#    CYTHON = True
+#except ImportError:
+#    CYTHON = False
+CYTHON = False
 
 cmdclass = {}
 ext_modules = []
